ui/lib/replication/addon/components/replication-overview-mode.hbs

```diff
@@ -0,0 +1,49 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<Hds::Card::Container @level="mid" @hasBorder={{true}} ...attributes>
+  <div class="has-padding-m">
+    <div class="flex">
+      <Icon @size="24" @name={{this.details.icon}} />
+      <Hds::Text::Display
+        @tag="h2"
+        @size="400"
+        data-test-overview-mode-title
+      >{{this.details.blockTitle}}</Hds::Text::Display>
+    </div>
+    <div class="has-top-padding-m has-bottom-padding-m" data-test-overview-mode-body>
+      {{#if (not (has-feature this.details.feature))}}
+        <Hds::Text::Body @size="300" @tag="div">
+          {{this.details.upgradeTitle}}
+          <ExternalLink @href={{this.details.upgradeLink}} class="link" data-test-upgrade-link={{@mode}}>
```

Comment:
Also just an FYI but I think we want to just  HDS links with `@isHrefExternal={{true}}` and move away from `ExternalLink`